dev/io.openliberty.microprofile.telemetry.1.1.internal/src/io/openliberty/microprofile/telemetry11/internal/rest/TelemetryClientBuilderListener.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry11.internal.rest;
+
+import static org.osgi.service.component.annotations.ConfigurationPolicy.IGNORE;
+
+import org.osgi.service.component.annotations.Component;
+
+import io.openliberty.restfulWS.client.ClientBuilderListener;
+import jakarta.ws.rs.client.ClientBuilder;
+
+/**
+ * Adds the Telemetry client filter when a rest client is built.
+ * <p>
+ * This gets called for both JAX-RS Client and MP Rest Client.
+ */
+@Component(configurationPolicy = IGNORE)
+public class TelemetryClientBuilderListener implements ClientBuilderListener {
```

Comment:
As a followup: this appears to contradict `TelemetryRestClientBuilderListener` which appears to handle MP Rest Client. Check which is correct and fix the comments.